pkg/controllers/job/job_controller_handler.go

```diff
@@ -134,15 +134,11 @@ func (cc *Controller) addPod(obj interface{}) {
 
 	jobName, found := pod.Annotations[vkbatchv1.JobNameKey]
 	if !found {
-		glog.Infof("Failed to find jobName of Pod <%s/%s>, skipping",
-			pod.Namespace, pod.Name)
```

Comment:
I would rather leave it here, and check the ownerreference of the pod at the begining.
Because this is so silent, we can not know what happened when unexpected result occurs.